site/homepage/content/release-notes.md

```diff
@@ -11,6 +11,9 @@ description = "Information about changes in recent Hono releases. Includes new f
 * When running in a Kubernetes cluster with nodes using cgroups v2, the 'hono.command_internal.*' Kafka topics were not
   being cleaned up. This has been fixed. Note that the solution requires the Hono protocol adapter pods to have
   a service account with an assigned RBAC role that allows to perform "get" on the "pods" resource.
+* When using Pub/Sub messaging, there were potentially issues concerning the AMQP connection between protocol adapter
+  and command router, leading for example to timeouts when MQTT devices subscribed/unsubscribed to the command topic.
+  This has been fixed.
```

Comment:
FMPOV we should also merge this into the 2.4.0 release. What do you think?

Reply:
Yes, after merging this PR, I'll apply the change also to the 2.4.x and 2.3.x branches.

Reply:
Good, do you than still want to list this as a fix for 2.5.0?

Reply:
This PR is for the master branch and therefore is for the upcoming 2.5.0 release. 
I've assigned #3574 also to the 2.4.1 and 2.3.2 release projects. I'll apply the change to the corresponding branches directly then, after this PR here got approved and merged.